doc/classes/TileMapLayer.xml

```diff
@@ -0,0 +1,309 @@
+		<method name="get_cell_tile_data" qualifiers="const">
+			<return type="TileData" />
+			<param index="0" name="coords" type="Vector2i" />
+			<param index="1" name="use_proxies" type="bool" default="false" />
+			<description>
+				Returns the [TileData] object associated with the given cell, or [code]null[/code] if the cell does not exist or is not a [TileSetAtlasSource].
+				If [param use_proxies] is [code]false[/code], ignores the [TileSet]'s tile proxies, returning the raw alternative identifier. See [method TileSet.map_tile_proxy].
+				[codeblock]
+				func get_clicked_tile_power():
+				    var clicked_cell = tile_map_layer.local_to_map(tile_map_layer.get_local_mouse_position())
+				    var data = tile_map_layer.get_cell_tile_data(clicked_cell)
+				    if data:
+				        return data.get_custom_data("power")
+				    else:
+				        return 0
+				[/codeblock]
+			</description>
+		</method>
+		<method name="get_coords_for_body_rid" qualifiers="const">
+			<return type="Vector2i" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the coordinates of the tile for given physics body [RID]. Such an [RID] can be retrieved from [method KinematicCollision2D.get_collider_rid], when colliding with a tile.
+			</description>
+		</method>
+		<method name="get_navigation_map" qualifiers="const">
+			<return type="RID" />
+			<description>
+				Returns the [RID] of the [NavigationServer2D] navigation used by this TileMapLayer. 
+				By default this returns the default [World2D] navigation map, unless a custom map was provided using [method set_navigation_map].
+			</description>
+		</method>
+		<method name="get_neighbor_cell" qualifiers="const">
+			<return type="Vector2i" />
+			<param index="0" name="coords" type="Vector2i" />
+			<param index="1" name="neighbor" type="int" enum="TileSet.CellNeighbor" />
+			<description>
+				Returns the neighboring cell to the one at coordinates [param coords], identified by the [param neighbor] direction. This method takes into account the different layouts a TileMap can take.
+			</description>
+		</method>
+		<method name="get_pattern">
+			<return type="TileMapPattern" />
+			<param index="0" name="coords_array" type="Vector2i[]" />
+			<description>
+				Creates a new [TileMapPattern] from the given set of cells.
+			</description>
+		</method>
+		<method name="get_surrounding_cells">
+			<return type="Vector2i[]" />
+			<param index="0" name="coords" type="Vector2i" />
+			<description>
+				Returns the list of all neighbourings cells to the one at [param coords].
+			</description>
+		</method>
+		<method name="get_used_cells" qualifiers="const">
+			<return type="Vector2i[]" />
+			<description>
+				Returns a [Vector2i] array with the positions of all cells containing a tile. A cell is considered empty if its source identifier equals [code]-1[/code], its atlas coordinates identifiers is [code]Vector2(-1, -1)[/code] and its alternative identifier is [code]-1[/code].
```

Comment:
```suggestion
				Returns a [Vector2i] array with the positions of all cells containing a tile. A cell is considered empty if its source identifier equals [code]-1[/code], its atlas coordinate identifiers is [code]Vector2(-1, -1)[/code], and its alternative identifier is [code]-1[/code].
```

Reply:
I removed the 's' to "identifiers"  instead, as coordinates should stay plural.

Reply:
Same here, it's attributive